src/rendering/RenderUtil.cc

```diff
@@ -457,6 +520,37 @@ void RenderUtil::Update()
       visual->SetUserData("temperature", temp.second);
     }
   }
+
+  // create new collision visuals
+  {
+    if (!newCollisionLinks.empty())
+      DeselectAllEntities();
+
+    for (auto const& link : newCollisionLinks)
+    {
+      std::vector<Entity> colEntities =
+          this->dataPtr->linkToCollisionEntities[link];
+
+      for (const auto& colEntity : colEntities)
+      {
+        if (!this->dataPtr->sceneManager.HasEntity(colEntity))
+        {
+          sdf::Collision collision = this->dataPtr->entityCollisions[colEntity];
+
+          sdf::Material material;
+          material.SetAmbient(math::Color(1, 0.5088, 0.0468, 1));
+          material.SetDiffuse(math::Color(1, 0.5088, 0.0468, 1));
```

Comment:
704af0dcc31636fa5f0a736752e55a1ec59bfd9a I set alpha to 0.7 because it was difficult to see at 0.5 (main visual color blended with collision color). Let me know if you would like me to update it to 0.5 instead